### PR TITLE
Create Contribuition Policy, Product Model Canvas and Sprint 1 Review correctly

### PR DESCRIPTION
**Pull request by luizguilherme5**
## Description

It was created the policy document and the product model canvas chart (which one had no issue related) and edit the information about the sprint review 1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/documentos/gerencia/pol_contribuicao.md [hunk 1]
```diff
@@ -3,9 +3,113 @@
...
+### Descrição
+* Utilize o _template_ de _issue_ destinada ao _pull request_.
+* Lembrando que o _pull request_ tem a _branch_ base a **_master_** e a _compare_ a branch que se deseja juntar.
+* **Lembrando**: assim que for realmente finalizado as alterações referentes ao _pull request_, deve-se retirar a tag **WIP**.
+
+![Imgur](https://i.imgur.com/REsu5YL.png)
+ 
+### _Reviewers_ 
+
+* Assinale os _reviewers_, ou seja, aqueles responsáveis à análise do _pull request_. Por exemplo, caso sua _feature_ esteja relacionada a arquitetura do projeto, assinale o **EPS** que desempenha esse papel.
```
Comment (gabibguedes) [inline docs/documentos/gerencia/pol_contribuicao.md]: EPS? Please, remove this part, change it to "member of the team".

**Review by gabibguedes — CHANGES_REQUESTED** [review]

File: docs/reunioes/sprints/sprint_1.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Reunião da Sprint 1
```
Comment (gabibguedes) [inline docs/reunioes/sprints/sprint_1.md:1]: Link this doc on the mkdocs.yml file, so it is accessible

**Review by gabibguedes — CHANGES_REQUESTED** [review]

**Review by renan601 — APPROVED** [review]

**Review by gabibguedes — APPROVED** [review]
Espetacular!

**Comment by gabibguedes** [comment]
link the mkdocs file to the sprint 1

**Pull request merged**